docs/introduction/getting-started.md

```diff
@@ -11,7 +11,7 @@ Nirvana currently uses .NET6.0. Please make sure that you have the most current
 ## Getting Nirvana
 
 ### Latest Release
-Contact the team to obtain the latest release.
+Contact the team to obtain the latest release and or Docker image.
```

Comment:
Should we add the email address you talked about today?

Reply:
yes, forgot to push changes